### DISCUSSION TITLE
Request for the display of the "Center of Mass" Auto/Custom in "Visible Collision Shapes"

### DISCUSSION THREAD

**Discussion by Conz3D** [godot-jolt/godot-jolt#650] [General]
It would be really helpful to see the center of mass in Debug -> "Visible Collision Shapes", Esp. for the "Auto Mode" CoM. But would be helpful for the Custom CoM Mode, too. 
Maybe the CoM  could also be visible in the Editor.

**Comment by mihe** [comment]
I have no control over what's displayed as part of "Visible Collision Shapes". In the case of the debug contact points that's simply [one part](https://github.com/godot-jolt/godot-jolt/blob/2e28ac63ffdd934f266f860c48985ab812062cf8/src/servers/jolt_physics_server_3d.cpp#L325-L335) of the physics server interface that I'm meant to implement and Godot itself takes care of the rest.

So you need to request this in [godotengine/godot-proposals](https://github.com/godotengine/godot-proposals) instead.
